packages/react-component-library/src/components/Select/Select.test.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import '@testing-library/jest-dom/extend-expect'
+import { render, RenderResult, fireEvent } from '@testing-library/react'
+
+import { Select } from '.'
+
+describe('Select', () => {
+  let wrapper: RenderResult
+  let onChangeSpy: (event: any) => void
```

Comment:
Can we be more explicit about that arg type?

Reply:
The prop on the interface is defined as below. We should not change this until we address all raised events in `v3`.

```javascript
onChange?: (event: any) => void
```